polkadot/primitives/src/runtime_api.rs

```diff
@@ -288,5 +288,10 @@ sp_api::decl_runtime_apis! {
 		/// Claim queue
 		#[api_version(11)]
 		fn claim_queue() -> BTreeMap<CoreIndex, VecDeque<ppp::Id>>;
+
+		/***** Added in v11 *****/
+		/// Elastic scaling support
+		#[api_version(11)]
```

Comment:
is it fine to reuse the api version? haven't seen this before

Reply:
Yeah, it is per method. Check
Above version 7 which added async backing.

Reply:
I guess this can be done only if there wasn't a runtime release in the meantime (between the claimqueue API and this API). Otherwise, versioning doesn't really make sense if we can add more things in a past version

Reply:
Yes it is only for staging APIs

Reply:
Currently production is at 10: https://github.com/polkadot-fellows/runtimes/blob/main/relay/kusama/src/lib.rs#L2147 

I'd  prefer to conflate all of the current staging methods into a runtime api version release (11). After discussing with @tdimitrov it seems there are no strict rules, except not adding on top of an released to prod version. 

@eskimor WDYT ?

Reply:
as discussed, sounds good. I thought the changes were already included in a testnet runtime release